### ISSUE TITLE
ENR-EYE: reduction in visual acuity

### ISSUE THREAD

**Issue by psergouniotis** [obophenotype/human-phenotype-ontology#3474]
further to discussions at the ERN-EYE ontology meeting 2017, this issue relates to refining the subclasses of the term "reduced visual acuity" (HP:0007663).

Current synonyms are: Central blurring of vision; Central visual loss; Decreased central vision; Decreased visual acuity; Poor visual acuity.

Consider removing the following from the synonym list:
*  "Central blurring of vision"; blurring of vision can be a synonym of Blurred vision (HP:0000622) and central can be added as a modifier. 
* "Central visual loss"; visual loss (HP:0000572) is an separate term and central can be added as a modifier. 

Current subclasses of  "reduced visual acuity" (HP:0007663) include:
* amblyopia
* sudden loss of visual acuity
* visual acuity test abnormality

"sudden loss of visual acuity" should be obsoleted and the content merged with "reduced visual acuity" (HP:0007663).

new subclasses should be added to "reduced visual acuity" (HP:0007663):

**Comment by psergouniotis** [comment]
as per ERN-EYE Low Vision and Patient Transverse Working Group [TWG-5] discussions (lead: David Keegan [@djkeegan71]) the following subclasses should be added to "Abnormal best corrected visual acuity test (HP:0030534)"; alternatively one could consider adding these directly as subclasses of "reduced visual acuity" (HP:0007663):

* mildly reduced visual acuity (syn mild reduction in visual acuity);
proposed definition: best corrected visual acuity between 0.21 and 0.59 logMAR (or logMAR equivalent).
*  moderately reduced visual acuity (syn moderate reduction in visual acuity);
proposed definition: best corrected visual acuity between 0.60 and 0.99 logMAR (or logMAR equivalent).
* severely reduced visual acuity (syn severe reduction in visual acuity);
proposed definition: best corrected visual acuity between 1.00 and 1.39 logMAR (or logMAR equivalent).
* very low visual acuity; 
proposed definition: best corrected visual acuity between 1.40 and 1.89 logMAR (or logMAR equivalent).
* Ultra-low vision;
proposed definition: best corrected visual acuity worse than 1.90 logMAR (or logMAR equivalent).

Please add the following subclasses to "Ultra-low vision"
    Motion Projection
    Light Projection
    Light Perception
    No Light Perception

**Comment by psergouniotis** [comment]
TWG-5 recommended that vision between -0.3 and +0.1 logMAR (or logMAR equivalent) should be considered normal (ie no reduction of visual acuity). This should be perhaps added in the definition of "Abnormal best corrected visual acuity test (HP:0030534).

**Comment by pnrobinson** [comment]
@psergouniotis -- I think these classes largely overlap with the new mild/moderate/severe vision loss classes. Maybe we should go over this again.

**Comment by psergouniotis** [comment]
There is an overlap but there are also important differences. 
One could have severe vision loss with normal visual acuity (e.g. when the visual field is very constricted). 

I feel that it would be helpful and appropriate to add the above terms that purely refer to visual acuity reduction. Please let me know if you would like any further information/clarification

**Comment by pnrobinson** [comment]
I think that we have a bunch of terms such as mild, moderate, severe vision loss
e.g.
https://hpo.jax.org/app/browse/term/HP:0001141
that are defined on the basis of visual acuity. Therefore, I would suggest that we change to primary label to those that are suggested above. Is this OK?
@psergouniotis

**Comment by psergouniotis** [comment]
I think mild/moderate/severe vision loss are slightly inaccurate terms. Definitely I would not keep them as primary labels and perhaps I would consider removing them and merging the content with:
- mildly reduced visual acuity
- moderately reduced visual acuity
- severely reduced visual acuity
respectively

**Comment by pnrobinson** [comment]
@psergouniotis
-- should we add the above terms or close this issue for now?

**Comment by psergouniotis** [comment]
hi Peter, what about these terms (as subclasses of ultra-low vision)
* "very low vision with retained motion projection": definition: very low vision but with retained ability to identify a moving object (typically hand motion at distance of 30 cm). 

* "very low vision with retained light projection": definition: very low vision but with retained ability to perceive the difference between light and dark. Also when light is projected in each of the four quadrants of the visual field, the individual is able to correctly identify the origin of the light stimulus.

* "very low vision with retained light perception": definition: very low vision but with retained ability to perceive the difference between light and dark.

* "very low vision with no light perception": definition: complete lack of light and form perception.

I could not find any formal definitions in the textbooks or internet so I tried to make my own.

**Comment by pnrobinson** [comment]
I am adding these terms but will use the label Ultralow instead of very low because of the label of the parent term. Your definitions will be the standard ones from now on!
